content/posts/book-log-on-combat.md

```diff
@@ -0,0 +1,37 @@
+---
+title: >-
+  「戦争の心理学」という本を読んだらだいたいのことは "極めて自然" に思えてきた
```

Comment:
@coderabbitai そういったエスケープを不要にするというのが https://github.com/pankona/pankona.github.com/pull/305 の目的で、実際にそのコミットを取り込んだ後はCIが通っています。 `The title contains double quotes which might cause YAML parsing issues. Properly escape the double quotes to avoid this issue.` ここは同意できますが、それによって Line Folding を ダブルクォートに戻すのは奇妙な提案では無いでしょうか。再考下さい

Reply:
@pankona 

![image](https://github.com/user-attachments/assets/e6cb361d-373d-429c-85b1-7733c250db6d)

ローカルで変更後の内容をサーブした限り良さそうです！